src/index.html

```diff
@@ -2,581 +2,592 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Furnitures shop</title>
-  <link rel="stylesheet" href="./css/style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Furnitures shop</title>
+    <link rel="stylesheet" href="./css/style.css">
 </head>
 
 <body>
-  <header>
+<header>
     <div class="top-bar">
-      <div class="container">
-        <div class="row">
-          <div class="col text-left">
-            <ul>
-              <li><a href="#">USD <i class="fas fa-caret-down"></i></a></li>
-              <li><a href="#">English <i class="fas fa-caret-down"></i></a></li>
-              <li><a href="#">Help <i class="fas fa-caret-down"></i></a></li>
-            </ul>
-          </div>
-          <div class="col text-right">
-            <ul>
-              <li><a href="#"><i class="fas fa-user"></i> Login</a></li>
-              <li><a href="#"><i class="fas fa-lock"></i> Register</a></li>
-              <li><a href="#"><i class="fas fa-bars"></i></a></li>
-            </ul>
-          </div>
+        <div class="container">
```

Comment:
you've probably changed indentation for this file. please restore the previous one.